### PR TITLE
{tools}[GCCcore/11.2.0] rocm-smi v4.5.0

### PR DESCRIPTION
**Pull request by nordmoen**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/rocm-smi/rocm-smi-4.5.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+docurls = ['https://rocmdocs.amd.com']
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ["https://github.com/RadeonOpenCompute/rocm_smi_lib/archive/"]
+sources = ["rocm-%(version)s.tar.gz"]
+
+checksums = ['43a2cc2ec548cc28698ca4fa01a947a4414febd433936a8d9770bf6f6ed55e4f']
+
+builddependencies = [
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/r/rocm-smi/rocm-smi-4.5.0-GCCcore-11.2.0.eb:20]:
```suggestion
builddependencies = [
    ('binutils', '2.37'),
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3108.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/050da20ec4fefe999468671180061368 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @nordmoen!

**Pull request merged**